assets/styles/base/_variables.scss

```diff
@@ -7,15 +7,22 @@ $default-font-size: 13px;
 $font-size-default: 13px;
 $font-size-small: 11px;
 $font-size-big: 15px;
+$font-size-bigger: 17px;
```

Comment:
I agree that naming could be improved however, the reasoning is to mirror DSM variables so it could be easier to integrate features in the future. [Naming](https://github.com/akeneo/pim-community-dev/blob/master/front-packages/akeneo-design-system/src/theme/common.ts#L105) is the same as in DSM's theme.